-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 142.384 sec <<< FAILURE! - in TestSuite
verifyingThePowerOfTwo(appium.wonkyCalculatorTests.tests.CalculatorPowerTests)  Time elapsed: 21.033 sec  <<< FAILURE!
java.lang.AssertionError: expected [Result: 12] but found [Result: 9]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at appium.wonkyCalculatorTests.tests.CalculatorPowerTests.verifyingThePowerOfTwo(CalculatorPowerTests.java:27)

